core/src/avm2/globals/xml.rs

```diff
@@ -456,9 +465,11 @@ pub fn descendants<'gc>(
     let multiname = name_to_multiname(activation, &args[0], false)?;
     let mut descendants = Vec::new();
     xml.node().descendants(&multiname, &mut descendants);
-    Ok(XmlListObject::new(activation, descendants, Some(xml.into())).into())
+    // FIXME: Usage of target property needs to be evaluated.
+    Ok(XmlListObject::new(activation, descendants, Some(xml.into()), None).into())
```

Comment:
I guess we need to verify what Flash does in practice, but in the spec I think this has a null [[TargetObject]] (so the current code is wrong) and no [[TargetProperty]] (implicit null?).

> 9.1.1.8 [[Descendants]] (P)
> 2. Let list be a new XMLList with list.[[TargetObject]] = null

Reply:
According to avmplus `XMLListObject::create` defaults to `null` target property and object (https://github.com/adobe/avmplus/blob/858d034a3bd3a54d9b70909386435cf4aec81d21/core/XMLListObject.h#L69C1-L69C1) and getDescendants does not specify either of those (https://github.com/adobe/avmplus/blob/858d034a3bd3a54d9b70909386435cf4aec81d21/core/XMLListObject.cpp#L192) so this is implicit null, yes.